mobile/native/src/logger.rs

```diff
@@ -168,7 +169,9 @@ pub fn init_tracing(level: LevelFilter, json_format: bool) -> Result<()> {
         .try_init()
         .context("Failed to init tracing")?;
 
-    tracing::info!("Initialized logger");
+    tracing::info!("Initialized tracing logger");
+
+    LogTracer::init()?;
```

Comment:
👍Nice! Should we do the same for the `coordinator`?

It's interesting because I could've sworn I've seen `rust-dlc` logs in the past (although there aren't many).